packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -582,6 +591,13 @@ public void setAdjustFontSizeToFit(boolean adjustsFontSizeToFit) {
     mAdjustsFontSizeToFit = adjustsFontSizeToFit;
   }
 
+  public void setFontSize(float fontSize) {
+    mFontSize =
+      mAdjustsFontSizeToFit
+        ? (float) Math.ceil(PixelUtil.toPixelFromSP(fontSize))
```

Comment:
Why are we using SP here?

Reply:
I ended up basing this code in the snippet from [TextAttributes.java](https://github.com/facebook/react-native/blame/90442ac66e202175fd25c09dd100f27ab3ac78dc/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributes.java#L131-L136).

Should it be removed?

Reply:
Oh, you're right.

> Can we reuse getEffectiveFontSize()?

Not directly because `ReactTextView` does not use `TextAttributes`. We could extract that logic and use a static method, but it would require passing `mMaxFontSizeMultiplier` and `mAllowFontScaling` (or just use defaults), which I'm not sure it would be the best, I'm not really sure what's the implications of this.

We could just remove the adjust logic and just use `toPixelFromDIP`, would it be best?